### DISCUSSION TITLE
bug on the filter to create view with boolean

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2030] [General]
![image](https://user-images.githubusercontent.com/3023526/126326091-747b5129-13e6-4cc1-a619-cd618b6e0290.png)
I have two rows one is "paye" at true and the other one is "paye" at false.

I created 2 views based on the boolean value of the "paye" attribute:
![image](https://user-images.githubusercontent.com/3023526/126326331-0d4d0a1a-2854-4861-acce-e6c3853cd9c8.png)

The view with the true filter is working but not the one with value false.

**Comment by mjashanks** [comment]
Yea - i see this now, I can recreate it. A workaround is to create the filter `paye notEquals true` !

I can make it happen by adding a Boolean column, after data already exists in the table. Underneath, the value defaults to `null`, so technically `null is not true`. We need to default it to false, or possibly ask for a default value when the column is added.

**Reply by oswaldB** [reply]
i think the problem is more global.

I changed the true/false by a 0 and 1. 

And even with 0 and 1 the filter is not working.

![image](https://user-images.githubusercontent.com/3023526/126327888-3fd4b3ee-e191-432e-8ab3-f3e8a0265425.png)

**Reply by mjashanks** [reply]
I have logged this here: https://github.com/Budibase/budibase/issues/2042 - thanks for reporting!
